.agent/rules/gemini.md

```
@@ -2,100 +2,235 @@
-uv run ruff format .
-# 自動修正可能なLintエラーの修正
-uv run ruff check --fix .
+### データ配置
+
+- **R2**: 正本（Parquet/Raw JSON、年月パーティショニング）
+- **DuckDB**: View レイヤー（`:memory:` で R2 直接クエリ）
+- **Qdrant**: 意味検索インデックス
+
+### コンポーネント依存
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)